crates/driver/src/domain/competition/order/mod.rs

```diff
@@ -399,6 +399,7 @@ pub struct Jit {
     pub sell_token_balance: SellTokenBalance,
     pub buy_token_balance: BuyTokenBalance,
     pub signature: Signature,
+    pub protocol_fees: Vec<FeePolicy>,
```

Comment:
Let's not give JIT orders protocol fees.
There usual use case is providing liquidity for a solution and is more akin to an AMM interaction which also doesn't get taxed in some way. Additionally protocol fee is something that should be determined by the protocol and at the moment the driver has no way of knowing whether a fee proposed by a solver for their JIT order is equal to whatever the backend would have decided.